.travis.yml

```diff
@@ -18,8 +18,8 @@ install:
 
 script:
   # Run all pytest unit tests
-  - python -m pytest -v tests --cov readabilipy --cov-report term-missing
+  - python -m pytest -v tests --cov readabilipy --cov-report term-missing --benchmark-disable
```

Comment:
I don't need the benchmarks to be run by travis, which they do automatically without this flag